### PR TITLE
Added Liquibase support, implemented soft delete concept for the Book…

### PR DESCRIPTION
**Pull request by AgroFix**
… entity, added delete, update methods

### CODE REVIEW

**Review by annton-shakur — APPROVED** [review]
Good job!

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 1]
```diff
@@ -8,9 +8,13 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Data
 @Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```
Comment (kshuryhin) [inline src/main/java/mate/academy/bookstore/model/Book.java]: 'org.hibernate.annotations.Where' is deprecated

File: src/main/java/mate/academy/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -32,4 +36,17 @@ public BookDto getBookById(@PathVariable Long id) {
...
         return bookService.createBook(createBookRequestDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        bookService.deleteBookById(id);
+    }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```
Comment (kshuryhin) [inline src/main/java/mate/academy/bookstore/controller/BookController.java]: Why do we use `NO_CONTENT` status for the **PUT** method?

**Review by kshuryhin — CHANGES_REQUESTED** [review]
Good job, but there are some points to consider

**Review by vovasalyha — APPROVED** [review]
LGTM

**Pull request merged**